build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
>The following set of APIs is supported in this release:
    Sequential streams (java.util.stream)
    A subset of java.time
    java.util.function
    Recent additions to java.util.{Map,Collection,Comparator}
    Optionals (java.util.Optional, java.util.OptionalInt and java.util.OptionalDouble) and some other new classes useful with the above APIs

bite me!